content/en/docs/setup/additional-setup/pod-security-admission/index.md

```diff
@@ -0,0 +1,87 @@
+
+{{< tip >}}
+    Namespaces in the mesh may also use the `restricted` [policy](https://kubernetes.io/docs/concepts/security/pod-security-standards/#baseline). You will need to configure the `seccompProfile` for your applications according to the policy specifications.
+{{< /tip >}}
+
+## Install Istio with PSA
+
+1. Create the `istio-system` namespace and label it to enforce the `privileged` policy.
+
+    {{< text bash >}}
+    $ kubectl create namespace istio-system
+    $ kubectl label --overwrite ns istio-system \
+        pod-security.kubernetes.io/enforce=privileged \
+        pod-security.kubernetes.io/enforce-version=latest
+    namespace/istio-system labeled
+    {{< /text >}}
+
+1. [Install Istio with CNI](/docs/setup/additional-setup/cni/#install-cni) on a Kubernetes cluster version 1.25 or later.
+
+    {{< text bash >}}
+    $ istioctl install --set components.cni.enabled=true -y
+    ✔ Istio core installed
+    ✔ Istiod installed
+    ✔ Ingress gateways installed
+    ✔ CNI installed
+    ✔ Installation complete
+    {{< /text >}}
+
+## Deploy the sample application
+
+1.  Add a namespace label to enforce the `baseline` policy for the default namespace where the demo application will run:
+
+    {{< text bash >}}
+    $ kubectl label --overwrite ns default \
+        pod-security.kubernetes.io/enforce=baseline \
+        pod-security.kubernetes.io/enforce-version=latest
+    namespace/default labeled
+    {{< /text >}}
+
+1. Deploy the sample application using the PSA enabled configuration resources:
+
+    {{< text bash >}}
+    $ kubectl apply -f @samples/bookinfo/platform/kube/bookinfo-psa.yaml@
+    service/details created
+    serviceaccount/bookinfo-details created
+    deployment.apps/details-v1 created
+    service/ratings created
+    serviceaccount/bookinfo-ratings created
+    deployment.apps/ratings-v1 created
+    service/reviews created
+    serviceaccount/bookinfo-reviews created
+    deployment.apps/reviews-v1 created
+    deployment.apps/reviews-v2 created
+    deployment.apps/reviews-v3 created
+    service/productpage created
+    serviceaccount/bookinfo-productpage created
+    deployment.apps/productpage-v1 created
+    {{< /text >}}
+
+1. Verify that the app is running inside the cluster and serving HTML pages by
+    checking for the page title in the response:
+
+    {{< text bash >}}
+    $ kubectl exec "$(kubectl get pod -l app=ratings -o jsonpath='{.items[0].metadata.name}')" -c ratings -- curl -sS productpage:9080/productpage | grep -o "<title>.*</title>"
+    <title>Simple Bookstore App</title>
+    {{< /text >}}
```

Comment:
Add a cleanup/uninstall section to remove the sample, remove labels from default ns, and uninstall Istio.